install.sh

```diff
@@ -24,7 +24,7 @@ SENTRY_EXTRA_REQUIREMENTS='sentry/requirements.txt'
 MINIMIZE_DOWNTIME=
 
 load_options() {
-  while [[ -n "$@" ]]; do
+  while [[ -n "${@:1}" ]]; do
```

Comment:
`${@:1}` is kind of a special case of substring expansion `${parameter:offset}`, I had to look this up myself. I'd suggest this to make it more readable, your choice though.

```suggestion
shift  # throw away $0
  while [[ -n "$@" ]]; do
```